kDrive/UI/Controller/Menu/Trash/TrashListViewModel.swift

```diff
@@ -307,6 +308,10 @@ class MultipleSelectionTrashViewModel: MultipleSelectionFileListViewModel {
                                                                  driveFileManager: driveFileManager) { [weak self] deletedFiles in
                 MatomoUtils.trackBulkEvent(eventWithCategory: .trash, name: "DeleteFromTrash", numberOfItems: selectedItemCount)
                 self?.removeFromRealm(realmConfiguration, deletedFiles: deletedFiles)
+
+                // quickwin for privacy, remove all image cache after a permanent clean
+                try? ImageCache.default.diskStorage.removeAll()
```

Comment:
Maybe we could also do it on "Empty thrash"

Reply:
Yes good idea, I just added that.